tools/requests_gun/internal/parse_args/parse_args.go

```diff
@@ -0,0 +1,40 @@
+package parse_args
+
+import (
+	"fmt"
+
+	flag "github.com/spf13/pflag"
+)
+
+type CliArgs struct {
+	Url         string
```

Comment:
Minor thing: the idiomatic way in Golang is to name acronyms in either all upper case or all lower case.